src/index.html

```diff
@@ -63,7 +63,7 @@
       $('.slider').slick({
         dots: true,
         infinite: true,
-        speed: 500,
+        speed: 1000,
```

Comment:
As for animation speed, 1000ms is a good value, but it is too slow for the slider.
Could you change this value back to 500, please?

Reply:
Done